kernel/packages/decentraland-loader/store/sceneSight/sceneSight.ts

```diff
@@ -0,0 +1,67 @@
+import { SceneId, StringCoordinate } from '../sceneInfo/sceneInfo'
+
+export type SceneSightState = Record<SceneId, number>
+
+export function increaseSightWithMap(state: SceneSightState, map: Record<SceneId, number>, keys?: string[]) {
+  if (!keys) {
+    keys = Object.keys(map)
+  }
+  const newState = { ...state }
+  for (let key of keys) {
+    if (!newState[key]) {
+      newState[key] = 0
+    }
+    newState[key] += map[key]
+    if (newState[key] === 0) {
+      delete newState[key]
+    }
+  }
+  return newState
+}
+
+export type SightMap = Record<StringCoordinate, SceneId>
+export type FlexibleSightMap = Record<SceneId, number>
+
+export function generateSightMap(inSight: StringCoordinate[], map: SightMap) {
+  const result: Record<SceneId, number> = {}
+  for (let parcel of inSight) {
+    if (map[parcel]) {
+      result[map[parcel]] = (result[map[parcel]] || 0) + 1
+    }
+  }
+  return result
+}
+
+export function sceneSightDelta(
+  from: FlexibleSightMap,
+  to: FlexibleSightMap,
+  keysFrom?: SceneId[],
+  keysTo?: SceneId[]
+) {
+  const result: {
+    lostSight: SceneId[]
+    newInSight: SceneId[]
+    currentSight: typeof to
+  } = {
+    lostSight: [],
+    newInSight: [],
+    currentSight: to
+  }
+  if (!keysFrom) {
+    keysFrom = Object.keys(from)
+  }
+  if (!keysTo) {
+    keysTo = Object.keys(to)
+  }
```

Comment:
How about removing these in favor of default values to parameters and avoid the lint warning:
```
export function sceneSightDelta(
  from: FlexibleSightMap,
  to: FlexibleSightMap,
  keysFrom?: SceneId[] = Object.keys(from),
  keysTo?: SceneId[] = Object.keys(to)
)
```